client/lib/setup/auth.coffee

```diff
@@ -22,7 +23,8 @@ setupPassport = ->
       headers: 'X-Access-Token': accessToken
       error: (m, err) -> done err
       success: (user) ->
-        id = user.get('channel_ids').concat(user.get('partner_ids'))[0]
+        partner_ids = jwtDecode(accessToken).partner_ids
```

Comment:
I don't think the client-side code actually has to be changed because you're already decoding the JWT and saving the partner ids on the server [here](https://github.com/artsy/positron/pull/1474/files#diff-2a1090257f616ec3ff944746d030cda6R58).

Reply:
Ooh yeah, you're totally right. This one is unnecessary. There is on client-side spot [here](https://github.com/artsy/positron/pull/1474/files#diff-ae3098d32d28ef9fa7bc649e32f5482fR48) that I think we will need to keep since it's where we check that the user stored in the session is in sync with the one stored in db.